qpc/source/utils.py

```diff
@@ -61,6 +61,8 @@ def build_source_payload(args, add_none=True):
         req_payload['hosts'] = args.hosts
     if hasattr(args, 'exclude_hosts') and args.exclude_hosts:
         req_payload['exclude_hosts'] = args.exclude_hosts
+    if hasattr(args, 'use_paramiko') and args.use_paramiko:
```

Comment:
since `args.use_paramiko` is a boolean, we want to check that it is not None here so it would be like: 
```
if hasattr(...) and args.use_paramiko is not None:
```